Apps/Playground/Win32/App.cpp

```diff
@@ -127,7 +127,7 @@ namespace
         std::string scriptsRootUrl = GetUrlFromPath(GetModulePath().parent_path().parent_path() / "Scripts");
 
         Babylon::ScriptLoader loader{*runtime};
-        loader.Eval("document = {}", "");
+        loader.Eval("document = {removeEventListener:function(){}}", "");
```

Comment:
I think we need to try to avoid making this hack worse. Can we file an issue?

Reply:
Yes! And I will remove this change in the app.